### DISCUSSION TITLE
Psram wird nicht erkannt

### DISCUSSION THREAD

**Discussion by jab42** [hoylabs/OpenDTU-OnBattery#1058] [Q&A] [answered]
Hallo,
ich habe mir ein S3 Board mit 8MB Psram gekauft.
Die 16Meg Flash werden erkannt, aber beim Psram steht 0.
Fehlt diese Option im Image?
Aufgespielt habe ich die neueste mit dem Webflasher.

**Comment by spcqike** [comment, marked as answer]
Ich kann mir vorstellen, dass der PSRAM aktuell noch gar nicht unterstützt und verwendet wird. Immerhin gibt es bisher auch keine offizielle Anforderung diesbezüglich.

**Reply by jab42** [reply]
Ich dachte halt wegen der Empfehlung zu dem 8 oder mehr Flash. die haben ja dann auch alle Psram.
Würde das schon gern testen...

**Comment by jab42** [comment]
Alles sehr seltsam. Ich habe jetzt ein tasmota s3 qio oio  auf ein 8/2 Modell gbrannt und er zeigt mir dann zwar die 8 Flash, aber weiter 5 psram (statt der 2 die eigentlich da sein sollten)...

**Comment by Solarteur** [comment]
Dieselbe Frage hier:

https://github.com/tbnobody/OpenDTU/discussions/1873

Antwort auch.

**Reply by jab42** [reply]
lustig.
es ist zwar theoretisch dieselbe frage, aber zu einer anderen hardware.
und damals hat die antwort auch nicht gepasst und nicht funktioniert.
lustigerweise passt sie aber jetzt zu dieser hardware hier. :-)
seltsam, dass ich das bei meiner suche nicht mehr gefunden habe... 
löst aber nicht wirklich das problem der falschanzeige...

**Comment by Tiese** [comment]
Ich wär' dann soweit...
![Bildschirmfoto 2024-06-21 um 15 48 47](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/141029037/0453737d-bcdd-4c8a-a563-b5544fbf7d92)
